src/collection-view.js

```diff
@@ -48,11 +48,6 @@ const CollectionView = Backbone.View.extend({
 
     monitorViewEvents(this);
 
-    this.once('render', this._initialEvents);
```

Comment:
fairly unrelated to this PR, but moving this to `render` with some of the other changes was a perf benefit